### DISCUSSION TITLE
Postgres!

### DISCUSSION THREAD

**Discussion by Norgul** [Sequel-Ace/Sequel-Ace#2091] [Ideas] [answered]
I know this is not a revolutionary idea, but I love Sequel Ace by far, from anything I've tried. Only thing which would make it even more perfect is a Postgres support. 

Is there even a remote possibility of integrating the support into it?

**Comment by Jason-Morcos** [comment, marked as answer]
https://github.com/Sequel-Ace/Sequel-Ace/discussions/1771
